app.py

```diff
@@ -1,19 +1,3 @@
-#!/usr/bin/env python3
-# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
im not sure if we can remove the copyright description, might need to add it for the newly added files